packages/Manager/src/main/resources/db/migration/V0044__water_allocation_by_area_add_ruamahanga.sql

```diff
@@ -0,0 +1,31 @@
+CREATE
+OR REPLACE VIEW water_allocations_by_area (area_id, allocation_amount) AS
+SELECT
+  area_id,
+  sum(allocation_plan) as allocation_amount
+FROM
+  water_allocations
+WHERE
+  effective_to is null
+GROUP BY
+  area_id
+
+UNION ALL
+
+SELECT
+'RuamahangaTotalSW' AS area_id
+, sum(allocation_plan) as allocation_amount
+FROM water_allocations
+WHERE area_id IN (
+'BoothsSW',
+'HuangaruaSW',
+'KopuarangaSW',
+'MangatarereSW',
+'PapawaiSW',
+'ParkvaleSW',
+'Ruamahanga_LowerSW',
+'Ruamahanga_MiddleSW',
+'Ruamahanga_UpperSW',
+'WaingawaSW',
+'WaiohineSW',
+'WaipouaSW')
```

Comment:
Being new to the data, I gather all of the RuamahangaTotalSW areas have an effective_to or something and won't be counted twice (unless they're meant to be)?

Reply:
This work adds this catchment management unit to the 'Allocations' section of the viewer, and you won't see it in the water use section. more work would be required to get it showing there.

there's a mapping which will link RuamahangaTotalSW to the broader catchment management unit, and those smaller areas will link to a sub unit, so a user will see both the sub unit and the broader unit when they hover on the map on the main page.